browser-test/src/application_navigation.test.ts

```diff
@@ -350,6 +350,60 @@ describe('Applicant navigation flow', () => {
       )
       await validateScreenshot(page, 'program-out-of-date')
     })
+
+    it('shows "no changes" page when a duplicate application is submitted', async () => {
+      const {page, applicantQuestions} = ctx
+      await applicantQuestions.applyProgram(programName)
+
+      // Fill out application and submit.
+      await applicantQuestions.answerDateQuestion('2021-11-01')
+      await applicantQuestions.answerEmailQuestion('test1@gmail.com')
+      await applicantQuestions.clickNext()
+      await applicantQuestions.clickNext()
+      await applicantQuestions.answerAddressQuestion(
+        '1234 St',
+        'Unit B',
+        'Sim',
+        'WA',
+        '54321',
+      )
+      await applicantQuestions.clickNext()
+      await applicantQuestions.answerRadioButtonQuestion('one')
+      await applicantQuestions.clickNext()
+      await applicantQuestions.answerPhoneQuestion(
+        'United States',
+        '4256373270',
+      )
+      await applicantQuestions.clickNext()
+      await applicantQuestions.submitFromReviewPage()
+
+      // Submit the application again without editing it
+      await applicantQuestions.returnToProgramsFromSubmissionPage()
+      await applicantQuestions.clickApplyProgramButton(programName)
+      await applicantQuestions.submitFromReviewPage()
+
+      // See the duplicate submissions page
+      await validateScreenshot(page, 'duplicate-submission-page')
```

Comment:
Can we verify that we're on the page by looking for some identifying text, like a header, as well?

The screenshot won't fail the test until it compares them at the end.